features/test/client/artprize.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Artprize demo
+  
+  Scenario: A person wants to visit the artprize demo
+    Given the server should start
+      And a client browser should start up
+      And a client should be able to visit /artprize
+      And a client should be able to see an element tagged [ng-app="blockforge"]
```

Comment:
Every demo will have its own set of tests.
This one is specifically checking the ArtPrize demo.